src/nvim/event/pty_process.c

```diff
@@ -33,13 +33,11 @@
 # include "event/pty_process.c.generated.h"
 #endif
 
-static const unsigned int KILL_RETRIES = 5;
-static const unsigned int KILL_TIMEOUT = 2;  // seconds
-
 bool pty_process_spawn(PtyProcess *ptyproc)
   FUNC_ATTR_NONNULL_ALL
 {
   Process *proc = (Process *)ptyproc;
+  assert(!proc->err);
   uv_signal_start(&proc->loop->children_watcher, chld_handler, SIGCHLD);
   ptyproc->winsize = (struct winsize){ptyproc->height, ptyproc->width, 0, 0};
   struct termios termios;
```

Comment:
This variable is always initialized the same way and will be used only for reading. Could be made static.